pytorch/inference/docker/1.11/py3/cu115/Dockerfile.gpu

```diff
@@ -213,12 +213,13 @@ RUN pip install --no-cache-dir -U \
     boto3 \
     "cryptography>3.2" \
     enum-compat==0.0.3 \
-    ipython==8.1.0 \
+    "ipython>=8.10,<9" \
     numpy==1.22.2 \
     "opencv-python>=4.6,<5" \
     packaging \
     "Pillow>=9.3.0" \
     "pyyaml>=5.4,<5.5" \
+    "nvgpu==0.9.0" \
```

Comment:
SM image is not release for cu115, so not making SM changes here, just ec2 changes